doc/measures_functions.md

```diff
@@ -321,6 +321,60 @@ SELECT OBS_GetMeta(
 ) FROM tablename
 ```
 
+## OBS_MetadataValidation(extent geometry, geometry_type text, metadata json, target_geoms)
+
+The ```OBS_MetadataValidation``` function peforms a validation check over
+the known issues using the extent, type of geometry and metadata we're going
+to use in the ```OBS_GetMeta``` function.
+
+#### Arguments
+
+Name | Description
+---- | -----------
+extent | A geometry of the extent of the input geometries
+geometry_type | The geometry type of the source data.
+metadata | A JSON array composed of metadata input objects.  Each indicates one desired measure for an output column, and optionally additional parameters about that column
+target_geoms | Target number of geometries.  Boundaries with close to this many objects within `extent` will be ranked highest.
+
```

Comment:
Yes, that would be great. For best practices, I want to make sure she is aware of any additions to Data Observatory content and code ;-)  I can help with the copyedit at the end, for sure.